schemas/chemotherapy.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "chemotherapy",
+  "description": "The collection of data elements describing the details of a chemotherapy treatment completed by a donor.",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "submitter_treatment_id",
+      "valueType": "string",
+      "description": "Unique identifier of the treatment, as assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
+      }
+    },
+    {
+      "name": "chemotherapy_drug_name",
+      "description": "Name of agent or drug administered to patient as part of the chemotherapy treatment regimen.",
+      "valueType": "string",
+      "restrictions": {
+        "required": true,
+        "codeList": [
+          "Allopurinol",
+          "Dronabinol",
+          "Sucralfate",
+          "Lapatinib"
+        ]
+      },
+      "meta": {
+        "core": true
+      }
+    },
+    {
+      "name": "chemotherapy_dosage_units",
+      "description": "Indicate units used to record chemotherapy drug dosage.",
+      "valueType": "integer",
```

Comment:
@rosibaj can we have integer and codelist together ??

Reply:
typo on my part - should be string